### PR TITLE
[IMP] im_livechat, website_livechat: introduce LivechatButtonView/livechat

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2883259

### CODE REVIEW

File: addons/im_livechat/static/src/legacy/widgets/livechat_button.js [hunk 1]
```diff
@@ -39,8 +39,7 @@ const LivechatButton = Widget.extend({
         });
 
         this._history = null;
-        // livechat model
-        this._livechat = null;
+
```
Comment (oomsveta) [inline addons/im_livechat/static/src/legacy/widgets/livechat_button.js:42]: unnecessary newline

File: addons/im_livechat/static/src/models/livechat_button_view.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
```
Comment (oomsveta) [inline addons/im_livechat/static/src/models/livechat_button_view.js:5]: extra newline

**Review by oomsveta — CHANGES_REQUESTED** [review]

File: addons/im_livechat/static/src/models/livechat_button_view.js (hunk 2 above)
Comment (alexkuhn) [reply addons/im_livechat/static/src/models/livechat_button_view.js]: Changes done

File: addons/im_livechat/static/src/legacy/widgets/livechat_button.js (hunk 1 above)
Comment (alexkuhn) [reply addons/im_livechat/static/src/legacy/widgets/livechat_button.js]: Changes done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/94090

**Pull request closed without merging**